lib/daru/io/importers/base.rb

```diff
@@ -56,6 +56,47 @@ def self.from(relation)
         def self.read(path)
           new.read(path)
         end
+
+        def self.parse_log(file,format)
```

Comment:
Why do you think it is appropriate to add method `parse_log`, related to a very specific task, to `Base` importer?.. All code related to some importer should be totally _local_ to this importer: either in `importers/rails_log.rb`, or in `importers/rails_log/request_log_analyzer_patch.rb` (the latter is preferable, as well as extracting patch to a module, instead of doing `class_eval`)

Reply:
`parse_log` is a helper method as it will also be used by apache and amazon_s3 importer. Making this part local will later require adding the same code for the remaining two importers.

Reply:
>  I feel the method is not that specific to be ineligible for the base class.

In fact, it is. The base class provides the common base _all_ importers need, not "some 3 of 20 importers may need that method, so we'll just drop it here". It requires specific external gem. If several different importers will need it, the module with this "patch" could be stored in, like `importers/shared/request_log_analyzer_patch.rb`, and required/included where it is necessary.